packages/overlays/stories/local-overlay-placement.stories.js

```diff
@@ -122,7 +124,8 @@ storiesOf('Local Overlay System|Local Overlay Placement', module)
         default:
           placement = 'top left';
       }
-      managePosition(element, target, { placement, position: 'absolute' });
+      action('position: ')(placement);
```

Comment:
usage of action is no longer needed... a simple `console.log` is enough and it will shop up in the action logger